storage/util_test.go

```diff
@@ -16,9 +17,21 @@ import (
 	chk "gopkg.in/check.v1"
 )
 
+var (
+	overwriteRec *bool
```

Comment:
I think it's a little easier to use if you delcare this as a `bool` instead of a `*bool` and use the [flag.BoolVar](http://godoc.org/flag#BoolVar) method later.